install_config/install/prerequisites.adoc

```diff
@@ -79,7 +79,13 @@ back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
 
 |External etcd Nodes
-|Minimum 20 GB hard disk space for etcd data.
+a|* Minimum 20 GB hard disk space for etcd data.
+* Consult
+link:https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations[Hardware
+Recommendations] to properly size your etcd nodes.
```

Comment:
I don't thinks it's a concern at this time.  My 0th concern is getting the information in the hands of those who need it, and it staying current, and this is the easiest way to do that.